shared/src/components/keyboard/slider/keyboard-slider.component.html

```diff
@@ -3,6 +3,8 @@
     [moduleConfig]="layer.modules"
     [keybindAnimationEnabled]="keybindAnimationEnabled"
     [capturingEnabled]="capturingEnabled"
+    [selectedKey]="selectedKey"
+    [selected]="selectedKey && selectedKey.layerId === index"
```

Comment:
Not sure we can do that as `this.selectedKey = { layerId: this.currentLayer, moduleId, keyId };` has the info of which is the selected key on each level and I have to pass down this info to determine the `selected` flag for each level _(level can be layer, module or key)_.